src/test/java/tests/AllSongsTests.java

```diff
@@ -0,0 +1,57 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.AllSongsPage;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class AllSongsTests extends BaseTest {
+    @Test
+    public void countSongsInPlaylist() {
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+        AllSongsPage allSongsPage = new AllSongsPage(driver);
+
+        loginPage.login();
+        //Assert.assertTrue(homePage.getUserAvatar().isDisplayed());
+
+        homePage.clickAllSongs();
+        //String songsURL = "https://qa.koel.app/#!/songs";
+        //Assert.assertEquals(driver.getCurrentUrl(), songsURL);

```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.